manage_rancher.py

```diff
@@ -61,6 +62,23 @@ def setup(main_cfg: dict, main_logger: logging.Logger) -> None:
                 logger.debug({"message": "Setting narrenv from environment",
                               "key": match.group(1), "value": os.environ[k]})
 
+# to do: figure out how to use method defined in app.py instead
```

Comment:
I know this is a dumb way of getting the version, because the method is already in app.py.  I tried a few things but they didn't work, so did this stupid thing.  I would like to figure out the correct way to do this before merging.